### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by ApoloIV** [lepoco/wpfui#520] [Ideas]
Would be cool if we had a discord server for the community, improving communication and feedback.
I offer myself to create it if you guys give me a green light!

Thanks for everything,
Apolo

**Comment by pomianowski** [comment]
As you wish
https://github.com/lepoco/wpfui/commit/c0c4d7d80f52ea4ed66520bb18637bab1f1d5785

[![Discord](https://img.shields.io/discord/1071051348348514375?label=discord)](https://discord.gg/AR9ywDUwGq)
